### PR TITLE
propagate secrets for maven test

### PR DESCRIPTION
**Pull request by zcgandcomp**
propagate secrets for maven test
correct name of deploy job

### CODE REVIEW

**Review by petrdvorak — APPROVED** [review]
Looks OK

File: .github/workflows/maven-deploy.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
 
   maven-deploy-public:
     if: ${{ github.event_name == 'workflow_dispatch' }}
-    name: Deploy to ${{ inputs.environment }}
+    name: Deploy by parameter
```
Comment (banterCZ) [inline .github/workflows/maven-deploy.yml:43]: Really? I would expect something like `oss.sonatype`

**Review by banterCZ — APPROVED** [review]
Looks good

File: .github/workflows/maven-deploy.yml (hunk 1 above)
Comment (zcgandcomp) [reply .github/workflows/maven-deploy.yml:43]: the jobs are for automated push - it goes to jfrog all the time and the second job is for manual deployment, so you can specify the target. I change name of jobs to be more suitable..

**Review by banterCZ — APPROVED** [review]

**Pull request merged**